mk/extbld/arch-embox-clang

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
Is this a copy or a symlink? `arch-embox-g++` is already a symlink to `gcc`, couldn't `clang` be too? Or may be `arch-embox-gcc` could be used, even if it wraps clang?